### PR TITLE
bert-scorer ending

### PR DESCRIPTION
**Pull request by SeBorgey**

### CODE REVIEW

**Review by Samoed — APPROVED** [review]

File: autointent/modules/scoring/_bert.py [hunk 1]
```diff
@@ -79,12 +80,21 @@ def fit(
     ) -> None:
         if hasattr(self, "_model"):
             self.clear_cache()
-
+        if not isinstance(labels[0], list) and isinstance(labels[0], str):
+            self._label_encoder = LabelEncoder()
+            encoded_labels = self._label_encoder.fit_transform(labels)
+            labels = encoded_labels.tolist()
```
Comment (voorhs) [inline autointent/modules/scoring/_bert.py]: из-за этой проверки mypy выдает ошибку

трекать ошибки mypy можно локально с помощью `make typing`

Comment (voorhs) [reply autointent/modules/scoring/_bert.py]: откуда там вообще строки могут возникнуть?

**Review by voorhs — CHANGES_REQUESTED** [review]
я попытаюсь разобраться почему интеграционные тесты падают

File: tests/assets/configs/multilabel.yaml [hunk 2]
```diff
@@ -24,6 +24,14 @@
     - module_name: sklearn
       clf_name: [RandomForestClassifier]
       n_estimators: [5, 10]
+    - module_name: bert
+      model_config:
+        - model_name: bert-base-uncased 
+        - model_name: distilbert-base-uncased
+      num_train_epochs: [2, 3]
+      batch_size: [8, 16]
+      learning_rate: [5e-5, 3e-5]
```
Comment (Samoed) [inline tests/assets/configs/multilabel.yaml]: @voorhs Они падают из-за указания learning rate, который надо конвертировать во float

**Comment by voorhs** [comment]
@SeBorgey я бы вмердживал. или ты еще собираешься что-то сюда коммитить?
